File: 기초 사고력문제/백준_5543_상근날드/상근날드 .java
import java.util.*;

public class Main{
    public static void main(String[] args){
        Scanner s = new Scanner(System.in);
        int min1 = 2000;
        for(int i =0; i<3; i++){
            int burger = s.nextInt();
            if(burger < min1) min1 = burger;
        }
        int min2 = 2000;
        for(int i =0; i<2; i++){
            int drink = s.nextInt();
            if(drink < min2) min2 = drink;
        }
        System.out.println(min1 + min2 - 50);
    }
}

File: 기초 사고력문제/백준_5543_상근날드/상근날드.kt
import java.util.*

fun main(args: Array<String>) {
    val s = Scanner(System.`in`)
    var min1 = 2000
    for(i in 0..2){
        var burger = s.nextInt();
        if(burger < min1) min1 = burger
    }
    var min2 = 2000
    for(i in 0..1){
        var drink = s.nextInt()
        if(drink < min2) min2 = drink
    }

    print("${min1 + min2 - 50}")
}